tests/cases/fourslash/typeToStringCrashInCodeFix.ts

```diff
@@ -3,4 +3,4 @@
 // @noImplicitAny: true
 //// function f(y, z = { p: y[
 
-verify.not.codeFixAvailable();
+verify.codeFixAvailable([]);

```

Comment:
`verify.not` appears to be idiomatic in this test framework, so I have no problem leaving it even if it doesn't add expressiveness.